### PR TITLE
View product backend

### PR DESCRIPTION
**Pull request by heegupark**
view-product-backend

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -7,12 +7,63 @@ const staticMiddleware = require('./static-middleware');
 const sessionMiddleware = require('./session-middleware');
 
 const app = express();
+const bodyParser = require('body-parser');
```
Comment (thebearingedge) [inline server/index.js]: This is available as `express.json()`

File: server/index.js [hunk 2]
```diff
@@ -7,12 +7,63 @@ const staticMiddleware = require('./static-middleware');
...
+        res.json(product);
+      }
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'An unexpected error occurred.'
+      });
+    });
+});
```
Comment (thebearingedge) [inline server/index.js]: This is not a part of the feature.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: server/index.js (hunk 1 above)
Comment (heegupark) [reply server/index.js]: Changed!

File: server/index.js (hunk 2 above)
Comment (heegupark) [reply server/index.js]: I though it required all GET features. Just removed!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**